### PR TITLE
Hay cap nhat commit

### PR DESCRIPTION
**Pull request by Taihocmot**
Cap nhat them fock

### CODE REVIEW

File: c.txt [hunk 1]
```diff
@@ -1,3 +1,5 @@
 Noi dung C
 
-Them noi dung C2
\ No newline at end of file
+Them noi dung C2
+
+Them noi dung cap nhat fock
```
Comment (Taihochai) [inline c.txt:5]: Them mot vao ten nua

**Review by Taihochai — COMMENTED** [review]
cap nhat nhanh

File: c.txt [hunk 2]
```diff
@@ -1,3 +1,5 @@
 Noi dung C
 
-Them noi dung C2
\ No newline at end of file
+Them noi dung C2
+
```
Comment (Taihochai) [inline c.txt]: xoa dong nay

File: c.txt [hunk 3]
```diff
@@ -1,3 +1,5 @@
 Noi dung C
 
-Them noi dung C2
\ No newline at end of file
+Them noi dung C2
```
Comment (Taihochai) [inline c.txt:3]: them vao mot dung nua

Comment (Taihocmot) [reply c.txt:3]: ok

**Pull request closed without merging**